actions/watch.ts

```diff
@@ -127,15 +131,49 @@ async function getTradeableOrderWithSignature(
   conditionalOrder: ConditionalOrder,
   contract: ComposableCoW
 ) {
+  const proof = conditionalOrder.proof ? conditionalOrder.proof.path : [];
+  const offchainInput = "0x";
+  contract.getTradeableOrderWithSignature;
+  const { to, data } =
+    await contract.populateTransaction.getTradeableOrderWithSignature(
+      owner,
+      conditionalOrder.params,
+      offchainInput,
+      proof
+    );
+
+  // await contract.provider
+  //   .estimateGas({
+  //     to: contract.address,
+  //     data,
+  //   })
+  //   .then((r) => {
+  //     console.log("[getTradeableOrderWithSignature:estimateGas] Success", r);
+  //   })
+  //   .catch((e) => {
+  //     console.error(
+  //       '[getTradeableOrderWithSignature:estimateGas] Error during "getTradeableOrderWithSignature" call: ',
+  //       e
+  //     );
+  //   });
+
+  console.log("[getTradeableOrderWithSignature] Simulate", {
+    to,
+    data,
+  });
```

Comment:
If the objective is to get the `calldata` so that it can be tested / simulated elsewhere, may I suggest using `encodeFunctionData` to produce the `calldata` that you desire for the simulation?

Reply:
No, sorry, this was a test I did debug the issue yesterday. I should have removed it. Im doing a follow-up with some cleanup and a small refactor and I will kill this